### DISCUSSION TITLE
How to enable auto-sync on native/Android?

### DISCUSSION THREAD

**Discussion by kolinger** [floccusaddon/floccus#1433] [Q&A] [answered]
I'm not sure if I'm just not seeing something but there is no way to enable auto syncing in native/Android app? 

I can't find auto-sync enable toggle thus my account is enabled=false and thus it will never auto-sync. Doesn't matter what sync interval I choose or if I make change of bookmarks. Sync is never triggered since it's always stopped on `if (!account.getData().enabled)` [here](https://github.com/floccusaddon/floccus/blob/v4.19.1/src/lib/native/NativeController.js#L141). The only way how I can sync is to use manual sync button.

I can see option to enable auto-sync in [NewAccount.vue](https://github.com/floccusaddon/floccus/blob/v4.19.1/src/ui/views/NewAccount.vue#L248) but with condition only for browser version. The only other presence of auto-sync enable toggle what I could find is in [AccountCard.vue/Overview.vue](https://github.com/floccusaddon/floccus/blob/v4.19.1/src/ui/components/AccountCard.vue#L138) but as far as I can see Overview.vue is not accessible on Android or I'm unable to find it.

I carefully looked through all options but I can't find how to enable auto-sync or how to access Overview.vue in native/Android version. I'm being blind and/or dumb or is this option actually missing?

Was auto-sync intentionally excluded from native versions? Why would that be? Since auto-sync is implemented for native and works fine on Android, just the enable toggle is missing.

**Comment by marcelklehr** [comment, marked as answer]
Hello @kolinger 

This is a nice catch. By default accounts on native start out always being enabled, but if you encounter an error, then there is no way to turn on auto-sync anymore.

**Reply by marcelklehr** [reply]
I will fix this.
